source/UniqueItemStuff.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BattleTech;
+using BattleTech.UI;
+
+namespace MechEngineer
+{
+    public partial class MechEngineerSettings
+    {
+        public UniqueItem[] Uniques =
+        {
+            //new UniqueItem {ItemPrefix = "emod_armor_", ReplaceTag = "Armor"},
+            //new UniqueItem {ItemPrefix = "emod_structure_", ReplaceTag = "Structure"},
+            new UniqueItem {ItemPrefix = "emod_engine_", ReplaceTag = "Engine Core"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_std_center", ReplaceTag = "Engine Shielding"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_center", ReplaceTag = "Engine Shielding"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_left", ReplaceTag = "Engine Shielding Left"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_right", ReplaceTag = "Engine Shielding Right"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_right", ReplaceTag = "Engine Shielding Right"},
+            new UniqueItem {ItemPrefix = "Gear_Gyro_", ReplaceTag = "Gyro"},
+            new UniqueItem {ItemPrefix = "Gear_Cockpit_", ReplaceTag = "Cockpit"},
+        };
+
+        public UniqueCategory[] UniqueCategories =
+        {
+            //new UniqueCategory {Tag = "Armor"},
+            //new UniqueCategory {Tag = "Structure"},
+            new UniqueCategory {Tag = "Engine Core"},
```

Comment:
allow ChassisLocations and ComponentType to be other filters to detect a category? or should we keep the dumb down logic here?

Reply:
i think game logic work there good. 

ps. need to check some part in other locations